spec/models/keywords_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# frozen_string_literal = true
+
+RSpec.describe Keyword, type: :model do
+  describe '#model' do
+    context 'when model is updated' do
+      it 'status reads failed' do
+        keyword = Fabricate(:keyword)
+
+        keyword.update_status(:failed)
+
+        expect(keyword.status).to eq('failed')
+      end
```

Comment:
Let's rename the context and the test so it's clearer; the context should describe the precondition (in this case the given param) and the test name (`it` block) should describe the expected effect on the instance

```suggestion
    context 'given status is failed' do
      it 'updates the status' do
        keyword = Fabricate(:keyword)

        keyword.update_status(:failed)

        expect(keyword.status).to eq('failed')
      end
    end
```

If you agree, please also update the following tests accordingly 🙌🏼

Reply:
Updated here https://github.com/aetheryna/google-scraper-ruby/pull/36/commits/5ccdd75db2c408cb66f542abf903918ab83b1750 o7